README.rst

```diff
@@ -91,21 +91,21 @@ Example usage
 
     # create a bus instance
```

Comment:
```suggestion
    # create a bus instance (using a context manager to call `Bus.shutdown()` after use)
```

Reply:
I added a similar comment but with "with" statement instead of context manager, because people might know about it from using files but not know the name "context manager".